lib/rss/converter.rb

```diff
@@ -69,7 +69,7 @@ def def_else_enc(to_enc, from_enc)
       def_iconv_convert(to_enc, from_enc, 0)
     end
 
-    def def_same_enc()
+    def def_same_enc
```

Comment:
```suggestion
    def def_same_enc()
```

Not in `Style/StringLiterals`.

I added a "local revert" suggestion using the GitHub feature for that.